src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/content/components/MethodologyAccordionSection.tsx

```diff
@@ -0,0 +1,148 @@
+  content: ContentSection<EditableContentBlock>;
+  sectionKey: ContentSectionKeys;
+  methodologyId: string;
+  index: number;
+  id: string;
+}
+
+const MethodologyAccordionSection = ({
+  sectionKey,
+  content,
+  methodologyId,
+  ...restOfProps
+}: MethodologyAccordionSectionProps) => {
+  const { isEditing } = useContext(EditingContext);
+  const { caption, heading, id: sectionId } = content;
+  const { content: sectionContent = [] } = content;
+  const [isReordering, setIsReordering] = useState(false);
+
+  const {
+    addContentSectionBlock,
+    deleteContentSectionBlock,
+    updateContentSectionBlock,
+    updateSectionBlockOrder,
+    updateContentSectionHeading,
+    removeContentSection,
+  } = useMethodologyActions();
+
+  const addBlockToAccordionSection = useCallback(() => {
+    addContentSectionBlock({
+      methodologyId,
+      sectionId,
+      block: {
+        type: 'MarkdownBlock',
+        order: sectionContent.length,
+        body: '',
+      },
+      sectionKey,
+    });
+  }, [methodologyId, sectionId, sectionContent.length, addContentSectionBlock]);
+
+  const updateBlockInAccordionSection = useCallback(
+    (blockId, bodyContent) => {
+      updateContentSectionBlock({
+        methodologyId,
+        sectionId,
+        blockId,
+        bodyContent,
+        sectionKey,
+      });
+    },
+    [methodologyId, sectionId, updateContentSectionBlock],
+  );
+
+  const removeBlockFromAccordionSection = useCallback(
+    (blockId: string) =>
+      deleteContentSectionBlock({
+        methodologyId,
+        sectionId,
+        blockId,
+        sectionKey,
+      }),
+    [methodologyId, sectionId, deleteContentSectionBlock],
+  );
+
+  const reorderBlocksInAccordionSection = useCallback(
+    order => {
+      updateSectionBlockOrder({
+        methodologyId,
+        sectionId,
+        order,
+        sectionKey,
+      });
+    },
+    [methodologyId, sectionId, updateSectionBlockOrder],
+  );
+
+  const onSaveHeading = useCallback(
+    (newHeading: string) =>
+      updateContentSectionHeading({
+        methodologyId,
+        sectionId,
+        heading: newHeading,
+        sectionKey,
+      }),
+    [methodologyId, sectionId, updateContentSectionHeading],
+  );
+
+  const removeSection = useCallback(
+    () =>
+      removeContentSection({
+        methodologyId,
+        sectionId,
+        sectionKey,
+      }),
+    [],
+  );
+
+  return (
+    <EditableAccordionSection
+      heading={heading}
+      caption={caption}
+      isReordering={isReordering}
+      headerButtons={
+        <Button
+          onClick={() => setIsReordering(!isReordering)}
+          className={`govuk-button ${!isReordering &&
+            'govuk-button--secondary'}`}
+        >
+          {isReordering ? 'Save section order' : 'Reorder this section'}
+        </Button>
+      }
+      {...content}
+      onHeadingChange={onSaveHeading}
+      onRemoveSection={removeSection}
+      {...restOfProps}
```

Comment:
Spread props should generally come first otherwise you can get accidental overriding of props. The only time you would want something like this would be if you're intentionally allowing consuming components to override specific props.